### ISSUE TITLE
It does not switch, but tries to mine with both miners. 

### ISSUE THREAD

**Issue by martin-karailiev** [DeroLuna/dero-miner#7]
With the lasted beta 2xthreads are received and the machine restarts. No auto algo switching under HiveOS

![image](https://github.com/DeroLuna/dero-miner/assets/105743088/d7f9b921-80e7-4ca3-a070-1f8004ed4ce0)

**Comment by AlienMiner303** [comment]
yep same here, thought id dual mine some dero & zeph again since its been a while and im getting the same issue, deroluna miner no longer pauses xmrig, it tries to mine both at the same time. even tried xmrigcc and same issue. only thing i was able to do was lower my threads for deroluna so i dont get LA issues. hopefully they will fix this
